src/System.Net.WebSockets.Client/tests/ClientWebSocketUnitTest.cs

```diff
@@ -64,8 +79,33 @@ public void ClientWebSocket_NoConnect_CloseOutputAsync_Ok()
         }
 
         [Fact]
-        public void ClientWebSocket_NoConnect_ReceiveAsync_Throws()
+        public void CloseAsync_CreateAndReceive_ThrowsInvalidOperationException()
         {
+            if (WebSocketHelper.ShouldSkipTestOSNotSupported(_output))
+            {
+                return;
+            }
+
+            var cws = new ClientWebSocket();
```

Comment:
As with Sockets: these tests use bad dev patterns to ensure that in certain cases the objects get GCed with no issues.
We don't expect FunctionalTests to be deterministic.